### DISCUSSION TITLE
Xugu database monitoring is not currently supported。

### DISCUSSION THREAD

**Discussion by YiTian-Sun** [apache/hertzbeat#1826] [General]
Of the same title, xugu database monitoring is not currently supported。

**Comment by tomsun28** [comment]
hi it seem xugu database is a database with a small users. I think the priority of implementing it is low, and its xugu jdbc code is closed-source, and there may cause license issues when use it.

**Reply by YiTian-Sun** [reply]
I think you can consider providing a user to upload the jdbc driver and define the connection configuration.  Like DBeaver.
